scripts/gulp/copy.js

```diff
@@ -2,14 +2,14 @@
 
 var gulp = require( 'gulp' );
 var fs = require('fs');
-var $ = require( 'gulp-load-plugins' )();
+var plugins = require( 'gulp-load-plugins' )();
```

Comment:
Minor: Remove the spacing in the parens here to keep it consistent with the rest of the file. We can update the entire project to our formatting standards at a later time.